src/api/java/baritone/api/schematic/FillSchematic.java

```diff
@@ -44,8 +43,6 @@ public BlockOptionalMeta getBom() {
     public IBlockState desiredState(int x, int y, int z, IBlockState current, List<IBlockState> approxPlaceable) {
         if (bom.matches(current)) {
             return current;
-        } else if (current.getBlock() != Blocks.AIR) {
-            return Blocks.AIR.getDefaultState();
```

Comment:
Do we know why this was originally here? IIRC this is what causes issues in higher versions of MC what with cave air and all that. Totally fine to remove if it fixes the related bug.

Reply:
ah, it's logandark code https://github.com/cabaletta/baritone/commit/991283182cde4b61bdef3743a1e2d53fc01c2bdb#diff-13b77f673395d72c1cda8da1c46c2613bc827c422cd6d0a7290b1816a31b8ca0R24

lol

Reply:
Well, I was hoping you knew why it was here.